src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue

```diff
@@ -246,6 +249,7 @@ export default {
 .related-resources {
 	&__header {
 		margin: 0 0 10px 46px;
+		color: var(--color-primary-element);
```

Comment:
We should not change the existing design, let's add an opt-in prop like https://github.com/nextcloud-libraries/nextcloud-vue/blob/c2095bfbaa9877a5920e7375c94f75390090e0ba/src/components/NcAppNavigationItem/NcInputConfirmCancel.vue#L81-L84 to conditionally make the header colored